src/components/AddForm/AddForm.driver.e2e.ts

```diff
@@ -21,3 +21,5 @@ export const driver = (page: any) => ({
   },
   enterPress: async () => enterPress(page),
 });
+
+export default driver;
```

Comment:
`export default` usually is not flexible enough

Reply:
disable this rule in config ?